admin/app/services/handle-error-response.js

```diff
@@ -0,0 +1,44 @@
+import every from 'lodash/every';
+import isEmpty from 'lodash/isEmpty';
+
+import { inject as service } from '@ember/service';
+import Service from '@ember/service';
+
+export default class HandleErrorResponseService extends Service {
+  @service notifications;
+
+  ERROR_MESSAGES = {
+    DEFAULT: 'Une erreur est survenue.',
+    STATUS_422: 'Cette opération est impossible.',
+    STATUS_404: 'Non trouvé.',
+  };
+
+  notify(anError, customErrorMessage) {
+    if (_isJSONAPIError(anError)) {
+      const { errors } = anError;
```

Comment:
**question**
Est-ce que ce serait pas plus approprié d'appeler le premier argument `aResponse` plutôt que `anError`. Ici on a une réponse qui contient des erreurs, ce qui est plus parlant qu'une erreur qui contient des erreurs. Et ça collerait avec le nom des méthodes qui handle des "errorResponse" et non des "error".

Reply:
On a catché une exception, c'est donc censé être une erreur mais on est pas sûr du format de cette erreur.
J'ai peur que juste "response" introduise une ambiguïté.
 Il y avait `errorResponse`  avant, 

`catch (errorResponse) {
      this._handleResponseError(errorResponse);
    }`
    
 est-ce que `errorResponse` ça serait mieux (ce serait redondant avec le nom de la méthode certes) ? 
Apparté : Mais même si on l'appelle `handleError` tout court, et qu'on passe 'error' en param, ça fera `handleError(error, customErrorMessage)`  ^^

Reply:
Je modifie ^^